docs/antrea-network-policy.md

```diff
@@ -1774,3 +1774,11 @@ Similar RBAC is applied to the ClusterGroup resource.
   matched at the same priority with conflicting actions. It will be the policy writer's
   responsibility to identify such ambiguities in rule definitions and avoid potential
   nondeterministic rule enforcement results.
+- Antrea has a higher-priority rule for the established connections which is expected to follow
+  the same "allow" action on the subsequent packets as how the first packet was processed.
+  An existing connection was set up before a drop/reject rule was realized, meaning it was
+  expected to "allow" such traffic when the connection was established. So the later valid packets
+  in this connection were also processed with "allow" action.
+  After traffic stopped, the existing connection record may be removed from conntrack cache
+  after a timeout interval, then the drop/reject rule will be applied to the new connection with
+  same properties.
```

Comment:
A translation from DeepL translator.  : )

Antrea has a higher priority rule for the established connections, which is expected to take the same "allow" action for subsequent packets as it did for the first packet processed. As a result, subsequent valid packets in that connection are also processed in an "allow" manner.
After traffic stops, the existing connection records may be deleted from the conntrack cache after a timeout, and then the delete/deny rule will be applied to new connections with the same attributes.

Reply:
Refactored some words.